### PR TITLE
epic: esl-core-2.0 basic updates

### PR DESCRIPTION
**Pull request by ala-n**
In bounds of this update:
- extend `CSSClassUtils` with `has` method
- add esl-utils with `attr` helpers
- extend `ESLBaseElement` with `$$attr` and `$$cls` shortcut
- update attributes decorators with ability to work on `Element` instances or objects with `{$host: Element}` representative

### CODE REVIEW

**Review by julia-murashko — APPROVED** [review]

File: src/modules/esl-utils/dom/attr.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export type AttributeTarget = Element | {$host: Element};
+
+/** @returns attribute ao fallback. Identical to getAttribute by default */
```
Comment (NastaLeo) [inline src/modules/esl-utils/dom/attr.ts:3]: typo: or

**Review by NastaLeo — APPROVED** [review]

**Review by Anna-MariiaPetryk — APPROVED** [review]

File: src/modules/esl-utils/dom/class.ts [hunk 2]
```diff
@@ -90,6 +99,15 @@ export abstract class CSSClassUtils {
     (state ? CSSClassUtils.add : CSSClassUtils.remove)(els, cls, locker);
   }
 
+  /**
+   * Check if all class from token string matches to the element pr elements.
```
Comment (dshovchko) [inline src/modules/esl-utils/dom/class.ts:103]:
```suggestion
   * Check if all class from token string matches to the element or elements.
```

**Review by dshovchko — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Pull request merged**